src/framework/constraints.ml

```diff
@@ -395,7 +395,8 @@ module WidenContextLifterSide (S:Spec)
 =
 struct
   module B = WidenContextLifter (S) (* can't just include this since type of ctx and some tf functions are different; TODO some generic functor to lift functions with conv, inj, proj? *)
```

Comment:
comment outdated?

Reply:
This comment is merged in from a recent change on master:
https://github.com/goblint/analyzer/commit/67c9c7dd96b030ea980a73dbc56af1c88bab4b96#diff-353a05f938a5047a918e48ea3a800661cd3a72304470b731f4d9bbe77bf0de96R397